examples/pedestrian_tracking/generate_reports.py

```diff
@@ -0,0 +1,100 @@
+import argparse
+import datetime
+import pandas as pd
+from pathlib import Path
+from fpdf import FPDF
+from core.common import utils
+
```

Comment:
This generation code looks good to me. But it is designed to applied on multi-edge inference and might not work for other bench, e.g., federated learning and lifelong learning. Our suggestion is to put the generate_report.py into ./multiedge_inference_bench/testreport.

BTW, the sample_report.pdf can also put in ./multiedge_inference_bench/testreport.

Reply:
Thanks for your suggestion, I have moved the code to the correct position.